### PR TITLE
Checks the deployment of 2 partition and 1 consumer (kafka,partition)

### PR DESCRIPTION
**Pull request by gicappa**
The CI is now testing that when helm performs an installation in the case of Kafka Partitioned it is possible to have a number of consumer different (<=) from the number of the partitions (i.e. partitions=2, consumers=1)

ref. Activiti/Activiti#3698

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -229,38 +229,130 @@ jobs:
...
+          export SSO_PROTOCOL=https
+          export GATEWAY_PROTOCOL=https
+          export GLOBAL_GATEWAY_DOMAIN=$CLUSTER_NAME.$CLUSTER_DOMAIN
+          export GATEWAY_HOST=gateway-$PREVIEW_NAME.$GLOBAL_GATEWAY_DOMAIN
+          export SSO_HOST=identity-$PREVIEW_NAME.$GLOBAL_GATEWAY_DOMAIN
+
+          make test/runtime-acceptance-tests
+
+      - name: Increase the consumers number and launch runtime acceptance test (only kafka partitioned)
+        if: ${{ matrix.messaging-broker == 'kafka' && matrix.messaging-partitioned == 'partitioned' }}
```
Comment (erdemedeiros) [inline .github/workflows/main.yml]: With the current configuration, we are running only the modeling tests for `rabbitmq-partitioned` and `kafka-non-partiotioned`. We need to have runtime acceptance tests running for those configurations as well.

**Review by erdemedeiros — CHANGES_REQUESTED** [review]
There is a problem with the current configuration. Check my comment below. Could you update the title of the PR as well?

File: .github/workflows/main.yml (hunk 1 above)
Comment (gicappa) [reply .github/workflows/main.yml]: Talking with @erdemedeiros we decided to modify the subject of the test of the code. So this comment is not applicable anymore.

**Review by erdemedeiros — APPROVED** [review]
LGTM!

**Pull request merged**